<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapViewer</class>
 <widget class="QWidget" name="MapViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Actions</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="qtDeleteGPSPointPushButton">
         <property name="text">
          <string>Delete GPS Point</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label1">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Latitude</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="qtLatitudeLineEdit">
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label3">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Longitude</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="qtLongitudeLineEdit">
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label2">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Altitude</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="qtAltitudeLineEdit">
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="qtGPSMapListView">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="tabKeyNavigation">
          <bool>true</bool>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DropOnly</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::MoveAction</enum>
         </property>
         <property name="movement">
          <enum>QListView::Static</enum>
         </property>
         <property name="selectionRectVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="qtSaveGPSDataPushButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Save GPS Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="qtConvertToLandmarkPushButton">
         <property name="text">
          <string>Convert To Landmark</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox1">
      <property name="title">
       <string>GPS</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QWidget" name="qtMapViewer" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="action_LoadWaypoints">
   <property name="text">
    <string>Load waypoints/GPS</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_SaveGPS">
   <property name="text">
    <string>Save as GPS data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_SaveCSV">
   <property name="text">
    <string>Save as CSV data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_ConvertToLandmarks">
   <property name="text">
    <string>Convert to Landmarks</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
